File: node-ex/src/entry.ts
import { ex_0, getFunc2Execute } from "./examples";

export function main() {
  if (process.argv.length < 3) {
    console.error(`No Number Args Passed`);
    process.exit(1);
  }

  const value = Number(process.argv[2])
  const func = getFunc2Execute(value);

  if (func === null) {
    console.error(`Example with number: ${value} does not exist`);
    process.exit(1);
  }

  func();
}

File: py-ex/entry.py
import sys
from dotenv import load_dotenv

# from examples.ex_0 import ex_0
from helpers.getExample import getExample

def main():

    if( len(sys.argv) < 3 ):
        print('Usage: python3 main.py <example_number>')
        exit(1)

    value = int(sys.argv[1])
    func = getExample( value )

    if( func == None ):
        print('Invalid Example Number')
        exit(1)

    load_dotenv(dotenv_path="../.env")
    func( sys.argv[2] )